.github/workflows/release_github.yml

```
@@ -0,0 +1,51 @@
+name: release_github
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  release:
+    # Only run this job for commits that indicate a release
+    if: startsWith(github.event.head_commit.message, 'chore(repo): release')
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 10: mapping values are not allowed in this context

(syntax-check)